src/main/java/core/basesyntax/Entry.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Entry<K, V> {
```

Comment:
Create class Node, it's better to make it inner class 
Don't use getters and setters in class Node

Reply:
not fixed

Reply:
you don`t need this class anymore